metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/Configuration.scala

```diff
@@ -0,0 +1,44 @@
+package it.pagopa.interop.metricsreportgenerator.util
+
+import cats.syntax.all._
+import scala.concurrent.Future
+import scala.util.Try
+import it.pagopa.interop.commons.cqrs.model.ReadModelConfig
+import it.pagopa.interop.commons.mail.MailConfiguration
+import it.pagopa.interop.commons.mail.MailConfiguration._
+import pureconfig._
+import pureconfig.generic.auto._
+import pureconfig.error.ConfigReaderException
+import javax.mail.internet.InternetAddress
+
+final case class Configuration(
+  environment: String,
+  recipients: List[InternetAddress],
+  readModel: ReadModelConfig,
+  collections: CollectionsConfiguration,
+  tokens: TokensBucketConfiguration,
+  storage: StorageConfiguration,
+  mailer: MailConfiguration
+)
+
+final case class CollectionsConfiguration(
+  maxParallelism: Int,
+  tenants: String,
+  agreements: String,
+  purposes: String,
+  eservices: String
+)
+final case class TokensBucketConfiguration(bucket: String, basePath: String)
+final case class StorageConfiguration(bucket: String, basePath: String)
+
+object Configuration {
+  implicit val overrideRecipientReader: ConfigReader[List[InternetAddress]] =
+    ConfigReader.fromStringTry[List[InternetAddress]](
+      _.split(',').map(_.trim).toList.traverse(s => Try(new InternetAddress(s)))
```

Comment:
here we can use the new function in [this PR](https://github.com/pagopa/interop-commons/pull/260), right?

Reply:
In that PR I use Either instead of Try.